README.md

```diff
@@ -1,16 +1,52 @@
 
 ![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
 
-# SuperTokens Website SDK
+# SuperTokens Javascript Frontend SDK
 
 <a href="https://supertokens.io/discord">
 <img src="https://img.shields.io/discord/603466164219281420.svg?logo=discord"
     alt="chat on Discord"></a>
-    
-## About
-This is a website / JavaScript SDK that is responsible for maintaining a SuperTokens session for website apps (SPA and non-SPA).
 
-Learn more at https://supertokens.io
+SuperTokens adds **secure login and session management** to your apps. This is the Javascript Frontend SDK for SuperTokens. [More SDKs are available](https://github.com/supertokens) for frontend and backend e.g. Node.js, Go, Python, React.js, React Native, Vanilla JS, etc.
+
+![Architecture Diagram](https://supertokens.com/img/architecture/self_hosted_generic.png)
+SuperTokens architecture is optimized to add secure authentication for your users without compromising on user and developer experience
+
+
+Learn more at [supertokens.com](https://supertokens.com)
+
+
+## How to install
+
+Using npm
+
+```
+npm i -s supertokens-website
+```
+
+OR simply add the following script tag to your HTML pages
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-website/bundle/bundle.js"></script>
+```
+
+## How to use
+
+1. Initialize SuperTokens in your frontend 
+```javascript
+supertokens.init({
+    apiDomain: "<URL to your auth backend>"
+});
+
+// Now your app will maintain secure SuperTokens sessions for your users
+```
+2. Make sure your backend has the needed auth functionalities
+> You can use one of the SuperTokens backend SDKs for this. 
+> Backend SDKs are available for
+> * [Node.js backend](https://github.com/supertokens/supertokens-node)
+> * [Golang backend](https://github.com/supertokens/supertokens-golang)
+> * [Python backend](https://github.com/supertokens/supertokens-python)
+
```

Comment:
Please also update the supertokens.io link to supertokens.com everywhere else in this PR (including for the email ID at the bottom)